src/test/java/teammates/test/cases/browsertests/FeedbackMsqQuestionUiTest.java

```diff
@@ -37,6 +43,7 @@ public void classSetup() {
     @Test
     public void allTests() throws Exception {
         testEditPage();
+        testStudentSubmitPage();
```

Comment:
This should not be done here. Use `StudentFeedbackSubmitPageUiTest` to test this.

Reply:
Just below that line of code is [this](https://github.com/TEAMMATES/teammates/blob/master/src/test/java/teammates/test/cases/browsertests/FeedbackMsqQuestionUiTest.java#L41) -

``` java
//TODO: move/create other MSQ question related UI tests here.
//i.e. results page, submit page.
```

That was why I wrote them here. Please clarify. Shall I move the student submission page tests to `StudentFeedbackSubmitPageUiTest`?

Reply:
Right, that. That comment was made before #3554 was opened. Sorry for the confusion; the preferred way is to be done in `StudentFeedbackSubmitPageUiTest`.